### DISCUSSION TITLE
迅雷网盘在线播放（已解决）

### DISCUSSION THREAD

**Discussion by Cyiming** [AlistGo/alist#1403] [Q&A]
我这能显示 但是无法播放 加载一会右下角感叹号

调用播放器也无法播放

**Comment by rakaki** [comment]
后台设置ua为 AndroidDownloadManager/11 (Linux; U; Android 11; M2004J7AC) ，然后开代理（proxy选项）,需要重启alist
这样设置就可以了

**Reply by Cyiming** [reply]
> 后台设置ua为 AndroidDownloadManager/11 (Linux; U; Android 11; M2004J7AC) ，然后开代理（proxy选项）,需要重启alist 这样设置就可以了

太厉害了 确实可以！！！

**Reply by WF1793456954** [reply]
请问alist在哪里设置UA？

**Comment by youtehub** [comment]
哈哈，确实是这个原因，会导致迅雷播放器无法调用到

**Reply by iMsazy** [reply]
为什么我这个就把迅雷打开了，然后让我下载。。。
